app/components/Analyst/Project/ProjectInformation/ProjectInformationForm.tsx

```diff
@@ -10,6 +10,66 @@ import { useCreateProjectInformationMutation } from 'schema/mutations/project/cr
 import ProjectTheme from 'components/Analyst/Project/ProjectTheme';
 import MetabaseLink from 'components/Analyst/Project/ProjectInformation/MetabaseLink';
 import Toast from 'components/Toast';
+import validateFormData from '@rjsf/core/dist/cjs/validate';
+import { Alert } from '@button-inc/bcgov-theme';
+
+const StyledAlert = styled(Alert)`
+  margin-bottom: 8px;
+  margin-top: 8px;
```

Comment:
```suggestion
p {
   margin-bottom: 0;
}
```

not a huge deal but there is still that weird margin inside the alert. either this or using a div or span instead of `<p>` would work.

Reply:
I am using a div now, I think the margin might be from the styling of the container for the content? That's my guess